src/app/shared/components/smart-search/smart-search.component.html

```diff
@@ -0,0 +1,34 @@
+<mat-form-field class="md-search" appearance="outline">
```

Comment:
The smart search above is for Role Namespace. Where is that part here?

Reply:
I don't quite understand the question. can add details
I use it for grouping and styling

Reply:
As discussed, the smart search for those 2 fields should only display list of role namespaces. Currently, the smart search component displays Orgs & Apps which is not relevant to Issuer Role & Restrictions fields.